### PR TITLE
changed functions and _targets.R file to incorporate swapping files i…

### PR DESCRIPTION
**Pull request by slevin75**
Swapped the file targets for each site into object targets and swapped the annotated_data object target to a file target.

image

### CODE REVIEW

File: 1_fetch/src/get_nwis_data.R [hunk 1]
```diff
@@ -31,14 +31,15 @@ download_nwis_site_data <- function(site_num,outdir, parameterCd, startDate, end
...
 
   # -- simulating a failure-prone web-sevice here, do not edit --
-  set.seed(Sys.time())
-  if (sample(c(T,F,F,F), 1)){
-    stop(site_num, ' has failed due to connection timeout. Try tar_make() again')
-  }
+ # set.seed(Sys.time())
+ # if (sample(c(T,F,F,F), 1)){
+ #   stop(site_num, ' has failed due to connection timeout. Try tar_make() again')
+ # }
```
Comment (jds485) [inline 1_fetch/src/get_nwis_data.R]: I think for this training this is supposed to remain uncommented.

**Comment by slevin75** [comment]
I thought so too but it kept giving me timeout errors when it was uncommented and I couldn't really figure out why.  It works fine when its commented out. I'm not sure I actually understand what this line of code is doing.

**Review by jds485 — APPROVED** [review]
Looks good! Everything compiled for me. You can merge when you're ready

**Comment by jds485** [comment]
> I thought so too but it kept giving me timeout errors when it was uncommented and I couldn't really figure out why. It works fine when its commented out. I'm not sure I actually understand what this line of code is doing.

This code is here just for the training to simulate a random server download error. It fails 25% of the time (when T is sampled)

**Pull request merged**